core/arch/arm/sta/sub.mk

```diff
@@ -1,5 +1,6 @@
 srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += sta_self_tests.c
 srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += core_self_tests.c
+srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += sta_debug.c
```

Comment:
`stats.c` would be short and to the point.
We may want to use a different config variable like `CFG_TEE_CORE_STATS` or so.